asterisk/rootfs/etc/cont-init.d/asterisk.sh

```diff
@@ -33,7 +33,7 @@ fi
 readonly ha_token="${HA_TOKEN:-"${SUPERVISOR_TOKEN:-}"}"
 
 readonly etc_asterisk="/etc/asterisk"
-readonly config_dir="/config/asterisk"
+readonly config_dir="/config"
```

Comment:
I am super in favor of this change, but this is a breaking change and should be  mentioned in the changelog. Or we can maybe make a script to migrate stuff automatically.

Reply:
Superseded-by https://github.com/TECH7Fox/asterisk-hass-addons/pull/366/commits/91c2843758f34ad59e8ee242be37588a2aa07a55.